map/src/login/DownloadBackupDialog.jsx

```diff
@@ -147,17 +155,22 @@ export default function DownloadBackupDialog({ openDownloadBackupDialog, setOpen
     }
 
     function prepareFileGroups(groups) {
-        let res = {};
+        let res = {
+            [menuType[0]]: null,
+            [menuType[1]]: null,
+            [menuType[2]]: null,
+            [menuType[3]]: null,
+        };
         let names = Object.keys(groups);
         names.forEach((n) => {
             if (userPlaces.includes(n)) {
-                res = addToGeneralGroup('My places', res, n, groups);
-            }
-            if (settings.includes(n)) {
-                res = addToGeneralGroup('Settings', res, n, groups);
-            }
-            if (resources.includes(n) || n.startsWith(`${FILE_TYPE}_`)) {
-                res = addToGeneralGroup('Resources', res, n, groups);
+                addToGeneralGroup([menuType[0]], res, n, groups);
```

Comment:
Very muddled if:

1) why indexes 0 1 3 2 instead of 0 1 2 3 were used?

2) no final else {} (what will happen when all if will got false)

Please think about clarifying this code.